pubspec.yaml

```diff
@@ -21,10 +21,10 @@ environment:
   sdk: ">=2.7.0 <3.0.0"
 
 dependencies:
+  firebase: ^7.3.0
```

Comment:
I don't use caret syntax - I know it's the recommended option so I'm not trying to get you to change but I'll give you my pitch as to why it's better to leave blank, which you're welcome to ignore :-)